quirks/password-rules.json

```diff
@@ -59,6 +59,9 @@
     "claimlookup.com": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: [@#$%^&+=!];"
     },
+    "clien.net": {
+        "password-roles": "minlength: 5; required: lower, upper; required: digit;"
```

Comment:
`password-roles` should be corrected to `password-rules`.

Reply:
> `password-roles` should be corrected to `password-rules`.
rebase done. thank you :)